docs/user-guide/garbage-collector.md

```diff
@@ -0,0 +1,21 @@
+---
+---
+
+* TOC
+{:toc}
+
+## WARNING: Garbage Collector is an alpha feature, use it at your own risk!
+
+### What does the garbage collector do
+
+The garbage collector (GC) monitors the cluster and checks the `metadata.ownerReferences` field of each object. If none of the owners present in `metadata.ownerReferences` exists in the cluster, the GC will request the API server to delete the object.
```

Comment:
When are these created?  What are these created on?  Where are they created (client-side, server side?)  Do these already exist, or are the new in 1.3?  Are these created only if GC is turned on?  How do I get GC for controllers that already exist?